src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java

```diff
@@ -74,7 +74,7 @@ public Integer getMessageId() {
 	public String getDescription() {
 		var description = "request from " + getUsername();
 
-		if (START_COMMAND.equals(message.text())) {
+		if (START_COMMAND.equalsIgnoreCase(message.text())) {
```

Comment:
Why did you change this check? The start command is triggered by Telegram when a user activates the bot and it will always be `/start`

Reply:
Just to be consistent with the equalsIgnoreCase already present in the code, but your point of view makes sense